harness/src/fuzz/firedancer.rs

```diff
@@ -235,6 +235,7 @@ fn parse_fixture_effects(
             .compute_budget
             .compute_unit_limit
             .saturating_sub(effects.compute_units_available),
+        return_data: None, // TODO: Omitted for now.
```

Comment:
Have you seen fixtures in the Firedancer test-vectors that test the return data for Stake? If so, we might want to add this to the Firedancer support here.

I can get a test going that actually runs against real test vectors to make sure this thing actually works, and you could rebase onto it if you want.

Reply:
Here's the PR: https://github.com/buffalojoec/mollusk/pull/65

Reply:
yea a lot of the fixtures contain expected return data for `GetMinimumDelegation`, i just havent looked through the fuzz and fd-fuzz code yet to see what support is needed since my usecase only depends on harness. will do that tho